examples/cmri_mpca/main.py

```diff
@@ -70,43 +70,69 @@ def main():
     y = landmark_df["Group"].values
     y[np.where(y != 0)] = 1  # convert to binary classification problem, i.e. no PH vs PAH
 
-    # plot the first phase of images
+    # plot the first phase of images with landmarks
+    marker_names = list(landmark_df.columns[1::2])
+    markers = []
+    for marker in marker_names:
+        marker_name = marker.split(" ")
+        marker_name.pop(-1)
+        marker_name = " ".join(marker_name)
+        markers.append(marker_name)
+
     if save_images:
-        visualize.plot_multi_images(
-            [images[i][0, ...] for i in range(n_samples)],
-            marker_locs=landmarks,
-            im_kwargs=dict(cfg.IM_KWARGS),
-            marker_kwargs=dict(cfg.MARKER_KWARGS),
-            n_cols=10,
-        ).savefig(str(save_images_location) + "/0)first_phase.png")
+        n_img_per_fig = 35
```

Comment:
Is 35 always the best number: would you consider make this part of the configuration in config yaml?